### PR TITLE
Re-adds tritonian gills

### PR DESCRIPTION
**Pull request by N8-Toe**
Adds back in my previous greatest code contribution. Tested locally and change log included

### CODE REVIEW

File: html/changelogs/N8gills.yml [hunk 1]
```diff
@@ -0,0 +1,37 @@
...
+#   spellcheck (typo fixes)
+#   experiment
+#   admin
+#################################
+
+# Your name.  
+author: N8-Toe
+
+# Optional: Remove this file after generating master changelog.  Useful for PR changelogs that won't get used again.
+delete-after: False
```
Comment (io-util) [inline html/changelogs/N8gills.yml]:
```suggestion
delete-after: True
```

**Review by io-util — CHANGES_REQUESTED** [review]
Just change the `delete-after` to make changelog generation a bit easier, besides that this looks fine.

**Comment by N8-Toe** [comment]
Request change done, Ready for re-review

also Merry Christmas

**Review by Superhats — CHANGES_REQUESTED** [review]
Your changelog file is improperly named. Make sure to follow the naming conventions in the readme;
it should be either [YOUR USERNAME]-PR-[YOUR PR NUMBER].yml or [YOUR USERNAME]-[YOUR BRANCH NAME]

**Comment by N8-Toe** [comment]
Changelog 2, Return of the log

**Review by Superhats — APPROVED** [review]
Godspeed, spaceman

**Pull request merged**